martinLi/assignments/cyclicSort/lc268.js

```diff
@@ -0,0 +1,44 @@
+//https://leetcode.com/problems/missing-number/
+
+/*
+
+Given an array nums containing n distinct numbers in the range [0, n],
+ return the only number in the range that is missing from the array.
+
+Follow up: Could you implement a solution using only O(1) extra space complexity 
+and O(n) runtime complexity?
+Input: nums = [3,0,1]// [1,0,3] // [0, 1,3]
+Output: 2
+Explanation: n = 3 since there are 3 numbers, so all numbers are in the range [0,3].
+ 2 is the missing number in the range since it does not appear in nums.
+Input: nums = [0,1]
+Output: 2
+Explanation: n = 2 since there are 2 numbers, so all numbers are in the range [0,2].
+2 is the missing number in the range since it does not appear in nums.
+*/
+var missingNumber = function (nums) {
```

Comment:
Would be great to see a high level overview and/or pseudocode outlining your approach.